data/src/generate_tsv_summary.py

```diff
@@ -11,7 +11,31 @@
 
 
 def _wiki_name_and_transcription_level(ele: List[str]) -> str:
-    return ele[3] + ele[5]
+    return ele[3] + ele[7]
+
+
+def _handle_modifier(
```

Comment:
Now that all files have scripts I guess we could grab the script more directly and just call this once, something like:
```
modifiers = language.get("dialect", {})
start = file_path.index("_") + 1
script_key = file_path[start:file_path.index("_", start)]
dialect_key = file_path[file_path.index("_", start) + 1: file_path.rindex("_")]
return language['script'][script_key], modifiers.get(dialect_key, "")
```
I'm not sure whether that's a worthwhile improvement though.

Reply:
+1.